src/utils/btc.ts

```diff
@@ -0,0 +1,83 @@
+import * as ecc from "@bitcoin-js/tiny-secp256k1-asmjs";
+import { initEccLib, address, networks } from "bitcoinjs-lib";
+import { NO_COORD_PK_BYTE_LENGTH } from "../constants/keys";
+
+// Initialize elliptic curve library
+export function initBTCCurve() {
+  initEccLib(ecc);
+}
+
+/**
+ * Check whether the given address is a valid Bitcoin address.
+ *
+ * @param {string} btcAddress - The Bitcoin address to check.
+ * @param {object} network - The Bitcoin network (e.g., bitcoin.networks.bitcoin).
+ * @returns {boolean} - True if the address is valid, otherwise false.
+ */
+export const isValidBitcoinAddress = (
```

Comment:
Given that simple-staking restricts the addresses that are available for use, should we do something here as well? My first guess is no, as we don't need to have any restrictions for the input address for the staking transaction.